tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -29,9 +29,9 @@ def _make_common_headers(auth_token, content_type):
 
     @staticmethod
     def _safe_to_log(server_response):
-        '''Checks if the server_response content is not xml (eg binary image or zip)
+        """Checks if the server_response content is not xml (eg binary image or zip)
         and and replaces it with a constant
```

Comment:
Since you're tweaking my comment, can you fix my double 'and and' while you're there :)